ManiaTouchControls/ManiaTouchControls/Objects/Player.c

```diff
@@ -31,6 +33,15 @@ bool32 Player_CanTransform(EntityPlayer *player)
     if (!SceneInfo->timeEnabled /*&& !ERZStart && (!PhantomEgg || PhantomEgg->disableSuperForm)*/)
         return false;
 
+#if RETRO_USE_MOD_LOADER
+    // Support for MegAmi's Super Cancel mod
+    ObjectERZSetup *ERZSetup = Mod.FindObject("ERZSetup");
+    bool32 superCancel = false;
+    Mod.LoadModInfo("SuperCancel", NULL, NULL, NULL, &superCancel);
+    if (superCancel && !ERZSetup && (player->state == Player_State_Transform || player->superState == SUPERSTATE_SUPER))
+        return true;
+#endif
+
```

Comment:
This is honestly fine (I don't see a better way of doing it,) just unwrap it from `RETRO_USE_MOD_LOADER` since everything here requires the mod loader anyway.